moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -282,6 +282,23 @@ export const ProjectsListViewFiltersConf = {
         "string_is_not_null",
       ],
     },
+    {
+      name: "project_tags",
+      label: "Project tags",
+      placeholder: "Project tags",
```

Comment:
@mddilley would you change the label + placeholder to "Tags"? That will match the way we have them labeled on the project summary tab.

Reply:
Definitely - just pushed this update ✅

Reply:
I'm dense, where is the placeholder on the UI? I thought it was in the advanced search input, but I don't see any text in those inputs.